.eslintrc

```diff
@@ -1,19 +1,67 @@
 {
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
+  ],
+  "plugins": [
+    "react"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
   "env": {
+    "es6": true,
     "browser": true,
     "node": true,
-    "es6": true
+    "jquery": true,
+    "mocha": true
```

Comment:
have you tried jest?

Reply:
I have not tried it yet, but I will take a look!